tests/unittest/aws_test.py

```diff
@@ -0,0 +1,379 @@
+FAKE_SNAPSHOT = aws.AWSSnapshot(
+    'fake-snapshot-id',
+    FAKE_VOLUME,
+    name='fake-snapshot')
+
+MOCK_DESCRIBE_INSTANCES = {
+    'Reservations': [{
+        'Instances': [{
+            'InstanceId': FAKE_INSTANCE.instance_id,
+            'Placement': {
+                'AvailabilityZone': FAKE_INSTANCE.availability_zone
+            },
+            'State': {
+                'Name': 'running'
+            }
+        }]
+    }]
+}
+
+MOCK_DESCRIBE_INSTANCES_TAGS = {
+    'Reservations': [{
+        'Instances': [{
+            'InstanceId': FAKE_INSTANCE.instance_id,
+            'Placement': {
+                'AvailabilityZone': FAKE_INSTANCE.availability_zone
+            },
+            'State': {
+                'Name': 'running'
+            },
+            'Tags': [{
+                'Key': 'Name',
+                'Value': FAKE_INSTANCE.name
+            }]
+        }]
+    }]
+}
+
+MOCK_DESCRIBE_VOLUMES = {
+    'Volumes': [{
+        'VolumeId': FAKE_VOLUME.volume_id,
+        'AvailabilityZone': FAKE_VOLUME.availability_zone,
+        'Attachments': []
+    }, {
+        'VolumeId': FAKE_BOOT_VOLUME.volume_id,
+        'AvailabilityZone': FAKE_BOOT_VOLUME.availability_zone,
+        'Attachments': []
+    }]
+}
+
+MOCK_DESCRIBE_VOLUMES_TAGS = {
+    'Volumes': [{
+        'VolumeId': FAKE_BOOT_VOLUME.volume_id,
+        'AvailabilityZone': FAKE_BOOT_VOLUME.availability_zone,
+        'Attachments': [{
+            'Device': '/dev/spf'
+        }],
+        'Tags': [{
+            'Key': 'Name',
+            'Value': FAKE_BOOT_VOLUME.name
+        }]
+    }]
+}
+
+MOCK_LIST_INSTANCES = {
+    FAKE_INSTANCE.instance_id: {
+        'region': FAKE_INSTANCE.region,
+        'zone': FAKE_INSTANCE.availability_zone
+    },
+    FAKE_INSTANCE_WITH_NAME.instance_id: {
+        'region': FAKE_INSTANCE_WITH_NAME.region,
+        'zone': FAKE_INSTANCE_WITH_NAME.availability_zone,
+        'name': FAKE_INSTANCE_WITH_NAME.name
+    }
+}
+
+MOCK_LIST_VOLUMES = {
+    FAKE_VOLUME.volume_id: {
+        'region': FAKE_VOLUME.region,
+        'zone': FAKE_VOLUME.availability_zone,
+    },
+    FAKE_BOOT_VOLUME.volume_id: {
+        'region': FAKE_BOOT_VOLUME.region,
+        'zone': FAKE_BOOT_VOLUME.availability_zone,
+        'name': FAKE_BOOT_VOLUME.name,
+        'device': '/dev/spf'
+    }
+}
+
+MOCK_CREATE_VOLUME = {
+    'VolumeId': 'fake-volume-from-snapshot-id',
+    'AvailabilityZone': FAKE_SNAPSHOT.availability_zone
+}
+
+MOCK_CREATE_SNAPSHOT = {
+    'SnapshotId': FAKE_SNAPSHOT.snapshot_id
+}
+
+MOCK_CALLER_IDENTITY = {
+    'UserId': 'fake-user-id'
+}
+
+
+class AWSAccountTest(unittest.TestCase):
+  """Test AWSAccount class."""
+
+  @mock.patch('libcloudforensics.aws.AWSAccount.ClientApi')
+  def testListInstances(self, mock_ec2_api):
+    """Test that instances of an account are correctly listed."""
+    mock_ec2_api.return_value.describe_instances.return_value = MOCK_DESCRIBE_INSTANCES  # pylint: disable=line-too-long
```

Comment:
You can use `pylint` statements at the class or function level, which might make it easier to read